.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+name: Node.js CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+        os: [windows-latest, ubuntu-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies for Viv
+        run: npm install
+      - name: Install dependencies for Avivator
+        run: npm run install:avivator
+      - name: Install dependencies needed for testing (only on Linux)
+        if: matrix.os == 'ubuntu-latest'
+        run: sudo apt-get install xvfb # needed for tape-run
+      - name: Test library (only on Linux)
+        if: matrix.os == 'ubuntu-latest'
+        run: xvfb-run --auto-servernum ./test.sh # for tape-run
+      - name: Build Avivator and Viv (only on Windows)
+        if: matrix.os == 'windows-laters'
+        run: npm run build && npm run build:avivator
```

Comment:
I would break this up into separate actions and remove all the `if` blocks. We are testing different things: Can viv build on windows, and can viv build and test on unbuntu.

```
jobs:
  build-windows:
   

  test:
```